merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/resources/SimulationResourceManager.java

```diff
@@ -0,0 +1,39 @@
+package gov.nasa.jpl.aerie.merlin.driver.resources;
+
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.jpl.aerie.merlin.protocol.types.RealDynamics;
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema;
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.Map;
+import java.util.Set;
+
+public interface SimulationResourceManager {
+  int DEFAULT_THRESHOLD = 1024;
```

Comment:
make `final` and drag down to `Streaming`, add a comment explaining what its for